runtime/pointers.ts

```diff
@@ -2254,7 +2254,10 @@ export class Pointer<T = any> extends Ref<T> {
         }
         //placeholder replacement
         if (Pointer.pointer_value_map.has(v)) {
-            if (this.#loaded) {throw new PointerError("Cannot assign a new value to an already initialized pointer")}
+            if (this.#loaded) {
+                console.log("value",v)
```

Comment:
Still necessary?

Reply:
Helps for debugging if this error occurs again